### PR TITLE
Create Chase transaction class parser

### PR DESCRIPTION
**Pull request by shuuuMONsta**

### CODE REVIEW

**Comment by jtran0** [comment]
Did you test this yet?

**Comment by shuuuMONsta** [comment]
Tested

File: transaction_parser/transaction.py [hunk 1]
```diff
@@ -6,12 +6,14 @@
 class Transaction:
     def __init__(
         self,
+        trasanction_date=" ",
```
Comment (jtran0) [inline transaction_parser/transaction.py:9]:
```suggestion
        trasanction_date="",
```

File: transaction_parser/transaction.py [hunk 2]
```diff
@@ -6,12 +6,14 @@
...
         posting_date=" ",
```
Comment (jtran0) [inline transaction_parser/transaction.py:10]:
```suggestion
        posting_date="",
```

File: transaction_parser/transaction.py [hunk 3]
```diff
@@ -6,12 +6,14 @@
...
         description=" ",
```
Comment (jtran0) [inline transaction_parser/transaction.py:11]:
```suggestion
        description="",
```

**Review by jtran0 — APPROVED** [review]

File: transaction_parser/chase_transaction_parser.py [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+            csvreader = csv.DictReader(statements)
+            for row in csvreader:
+                # Create a Transaction object using keyword assignments with default values
+                transaction_obj = transaction.Transaction(
+                    trasanction_date=row.get("Transaction Date", ""),
+                    posting_date=row.get("Posting Date", ""),
+                    description=row.get("Description", ""),
+                    amount=float(row.get("Amount", 0.0)),
+                    balance=float(row.get("Balance", 0.0)),
+                    category=row.get("Category", " "),
```
Comment (jtran0) [inline transaction_parser/chase_transaction_parser.py:21]:
```suggestion
                    category=row.get("Category", ""),
```

Comment (shuuuMONsta) [reply transaction_parser/chase_transaction_parser.py]: removed

File: transaction_parser/transaction.py [hunk 5]
```diff
@@ -6,12 +6,14 @@
...
         amount=0.0,
         balance=0.0,
         category=" ",
```
[the remark this answers is not in the document]
Comment (shuuuMONsta) [reply transaction_parser/transaction.py]: removed

File: transaction_parser/transaction.py (hunk 3 above)
Comment (shuuuMONsta) [reply transaction_parser/transaction.py]: removed

File: transaction_parser/transaction.py (hunk 1 above)
Comment (shuuuMONsta) [reply transaction_parser/transaction.py]: removed

File: transaction_parser/transaction.py (hunk 2 above)
Comment (shuuuMONsta) [reply transaction_parser/transaction.py]: removed

**Pull request merged**